lib/rules/disallow-multiple-var-decl.js

```diff
@@ -1,21 +1,23 @@
 /**
  * Disallows multiple `var` declaration (except for-loop).
  *
- * Types: `Boolean` or `String`
+ * Types: `Object`
```

Comment:
Well, you can still use it if you want, but the doc now says you can already start to use the new syntax. I'm open to changing it back anyway if you want :)